LibraBFT/Yasm/Properties.agda

```diff
@@ -272,6 +272,26 @@ module LibraBFT.Yasm.Properties
      ...| mws'' , vpb'' rewrite sym (msgSameSig mws) = MsgWithSig∈-++ʳ mws'' , vpb''
 
 
+     newMsg⊎msgSent4 :  ∀ {e pk v m pid sndr s' outs} {st : SystemState e}
+                   → (r : ReachableSystemState st)
+                   → (stP : StepPeerState pid (availEpochs st) (msgPool st)
+                                          (Map-lookup pid (peerStates st)) (s' , outs))
+                   → Meta-Honest-PK pk → (sig : WithVerSig pk v)
+                   → v ⊂Msg m → (sndr , m) ∈ msgPool (StepPeer-post (step-honest stP))
+                   → (m ∈ outs × ValidSenderForPK (availEpochs st) v pid pk
+                      × ¬ (MsgWithSig∈ pk (ver-signature sig) (msgPool st)))
+                     ⊎ MsgWithSig∈ pk (ver-signature sig) (msgPool st)
+     newMsg⊎msgSent4 {_} {pk} {v} {m} {pid} {sndr} {outs = outs} r stP pkH sig v⊂m m∈post
+       with Any-++⁻ (List-map (pid ,_) outs) m∈post
+     ...| inj₂ m∈preSt = inj₂ (mkMsgWithSig∈ m v v⊂m sndr m∈preSt sig refl)
+     ...| inj₁ nm∈outs
+       with Any-map (cong proj₂) (Any-map⁻ nm∈outs)
+     ...| m∈outs
+       with sps-avp r pkH stP m∈outs v⊂m sig
+     ... | inj₁ newVote = inj₁ (m∈outs , newVote)
```

Comment:
Please remove spaces between `...` and `|`.

I also prefer the `w` of `with` to align with the `|` on the following line, but I see we haven't been consistent about that everywhere and of course it's really minor.

Reply:
I did that and will do from now on, but you are right we haven't been consistent.